src/main/java/com/example/demo/controller/Error/ErrorExceptionHandler.java

```diff
@@ -0,0 +1,32 @@
+package com.example.demo.controller.Error;
+
+import org.springframework.dao.DataAccessException;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.sql.SQLIntegrityConstraintViolationException;
+
+@RestControllerAdvice
+public class ErrorExceptionHandler {
+
+    @ExceptionHandler(org.springframework.dao.EmptyResultDataAccessException.class)
+    public ResponseEntity<?> EmptyResultDataAccessException(EmptyResultDataAccessException e){
```

Comment:
제네릭에 와일드카드를 선언한 이유가 있나요?
```suggestion
    public ResponseEntity<ErrorResponse> EmptyResultDataAccessException(EmptyResultDataAccessException e){
```

Reply:
와일드카드가 지양되야 하는 것을 모르고 사용했습니다.